posts/urls.py

```diff
@@ -1,9 +1,9 @@
 from django.urls import path
-from .views import UserView, PostView, PostViewID, CommentViewList
+from .views import UserView, PostView, PostViewID, FrontEndPostViewID, CommentViewList
 
 urlpatterns = [
     path('users/', UserView.as_view(), name='users'),
     path('posts/', PostView.as_view(), name='posts'),
-    path('posts/<pk>', PostViewID.as_view(), name='postid'),
+    path('posts/<pk>', FrontEndPostViewID.as_view(), name='postid'),
```

Comment:
You should keep 
`path('posts/<pk>', PostViewID.as_view(), name='postid'),`
and add another one 
`path('frontend/posts/<pk>', FrontEndPostViewID.as_view(), name='postid'),` to keep the api end point consistent